action.yml

```diff
@@ -17,14 +10,91 @@ inputs:
     required: true
     default: ''
     description: 'The password of your builder account.'
-  mfaCode:
-    required: false
+  configVdf:
+    required: true
+    default: ''
+    description: 'The contents of STEAM_HOME/config/config.vdf.'
+  ssfnFileName:
+    required: true
+    default: ''
+    description: 'The basename of the STEAM_HOME/ssfn file.'
+  ssfnFileContents:
+    required: true
+    default: ''
+    description: 'The contents of the file at STEAM_HOME/ssfnFileName.'
```

Comment:
Do we still need this?

Reply:
yep, we need a way to pass the GitHub Secrets into the action

Reply:
Ah I see. I didn't realise we needed both the name of the file and the contents. Thought it would be either or because usually no information in conveyed in a file name.